scss/mixins/_forms.scss

```diff
@@ -49,7 +49,7 @@
 
       .form-control-color {
         @include form-validation-state-selector($state) {
-          width: add($form-color-width, $input-height-inner);
+          width: calc($form-color-width + $form-feedback-icon-size + .8125rem); // stylelint-disable-line function-disallowed-list
```

Comment:
I haven't found any logical calculation to avoid having `.8125rem` and use a Sass variable instead. If you have any ideas, don't hesitate.